script/Arcueid/Day 2/02_00_ARC02_1.txt

```diff
@@ -74,7 +74,7 @@ Undeterred and unexisting, you are dreaming of a story that has already passed.
 [sha:cfe91652b93028e1cc2070298a745a152b90c9e5]{
 -- Page 5, Offset 2609.
 -- <欠けていく月のように|貴方の影を求めるように>。
-<Much as the moon wanes|M u c h   a s   I   y e a r n   f o r   y o u r   s h a d o w>.
+<Much as the moon wanes|M u c h   a s   I   y e a r n   f o r   y o u r   s h a d o w          >.
```

Comment:
This one looks like this btw. Is that ok or should the ruby align with the text below? ![image](https://github.com/Tsukihimates/Tsukihime-Translation/assets/90473809/94e0d830-4f76-4ae3-b1e9-88bb937e3837)

Reply:
Better? ![image](https://github.com/Tsukihimates/Tsukihime-Translation/assets/90473809/65a50590-3bde-49c8-9286-557d48c3aaa2)